build.gradle

```diff
@@ -2142,6 +2145,15 @@ task releasePlugin(dependsOn: [buildPlugin, buildUpmPlugin]) {
         it.path + "/**/*"
       }
     }
+    delete fileTree(dir: project.ext.pluginUpmDir) 
```

Comment:
Could you add a comment about what this logic is doing?

Reply:
Done!